### DISCUSSION TITLE
Desktop app like Inntertune

### DISCUSSION THREAD

**Discussion by christian179717** [z-huang/music#612] [Q&A] [answered]
Hello,

I'm completely happy with Innertune.
Does anybody know an innertune-like app for linux? So without registration, add, own playlists, a big range of music (Spotify or YouTube) and the possibility of downloads?
I've already tried much but there was no program I was happy with.

**Comment by nikhilCad** [comment]
https://github.com/amd64fox/SpotX

**Reply by christian179717** [reply]
> without registration

**Comment by Lurux** [comment]
I currently Freetube and am fine with it, although it doesn't integrate with downloaded content (it's actually a "normal" Youtube client, more akin to Newpipe).

You may find other ideas [here](https://flathub.org/apps/search/youtube).

**Comment by TheTrashCrafter** [comment, marked as answer]
Audiotube on Linux and maybe hyperpipe on windows (it's a website though)

**Comment by ankit98singh** [comment]
@z-huang should definitely think of porting the current app to PC as I'm in love with the app design and feature set, if only you could have it for pc. Plllssss.... make one for windows & linux as well

**Comment by selurvedu** [comment]
It's not a proper replacement, but you can run InnerTune in Waydroid, even if you're on Xorg.
